beacon-chain/core/state/state.go

```diff
@@ -78,9 +77,9 @@ func GenesisBeaconState(
 		}
 
 		validatorRegistry[i] = validator
-
 	}
 
+	latestBalances := make([]uint64, len(genesisValidatorDeposits))
```

Comment:
I made this change for readability since seeing latestBalances next to the for loop made me think its being used there when it's not.